docs/manual-CN/README.md

```diff
@@ -118,7 +118,7 @@
   * [Graphd 配置](3.build-develop-and-administration/3.configurations/4.graph-config.md)
   * [Storaged 配置](3.build-develop-and-administration/3.configurations/5.storage-config.md)
   * [命令行终端配置](3.build-develop-and-administration/3.configurations/6.console-config.md)
-  * [Kernel 配置](3.build-develop-and-administration/3.configurations/6.console-config.md)
+  * [Kernel 配置](3.build-develop-and-administration/3.configurations/7.kernel-config.md)
```

Comment:
if you change the file name, please update the file name [here](https://github.com/vesoft-inc/nebula-docs-cn/blob/master/mkdocs.yml) in the `yaml` file.
![image](https://user-images.githubusercontent.com/42762957/83471608-b2906780-a4b7-11ea-8737-b9a0a67c7bfe.png)